scripts/common.sh.in

```diff
@@ -414,3 +404,22 @@ retry() {
 
     echo "$stdout"
 }
+
+check_minion_id() {
```

Comment:
Minor: Maybe just a small comment here to said that for expansions the check is automatically done by k8s node name validation

Reply:
Done